src/Foundatio/Storage/FolderFileStorage.cs

```diff
@@ -157,7 +177,8 @@ public class FolderFileStorage : IFileStorage {
             
             try {
                 File.Delete(Path.Combine(Folder, path));
-            } catch (Exception) {
+            } catch (Exception ex) when (ex is FileNotFoundException || ex is DirectoryNotFoundException) {
+                _logger.LogError(ex, "Error trying to delete file: {Path}.", path);
```

Comment:
I think this should be debug level. This isn't really an error you are trying to delete a file that doesn't exist.